Countly.js

```diff
@@ -199,13 +200,13 @@ Countly.sendEvent = function (options) {
  * Supported data type for segments values are String, int, double and bool
  * @return {String || void} error message or void
  */
-Countly.recordView = async function (recordView, segments) {
+Countly.recordView = function (recordView, segments) {
```

Comment:
is it a breaking change? what happened if we add await to non async function?
for e.g 
`await Countly.recordView('VIEW_NAME');`

Reply:
I have checked it:
it just shows a warning "'await' has no effect on the type of this expression"